diff --git a/ggml/src/ggml-vulkan/CMakeLists.txt b/ggml/src/ggml-vulkan/CMakeLists.txt
index af951a2..77f3e99 100644
--- a/ggml/src/ggml-vulkan/CMakeLists.txt
+++ b/ggml/src/ggml-vulkan/CMakeLists.txt
@@ -13,6 +13,11 @@ if (DEFINED ENV{VULKAN_SDK})
 endif()
 find_package(SPIRV-Headers CONFIG REQUIRED)
 
+if (TARGET SPIRV-Headers::SPIRV-Headers)
+    get_target_property(SPIRV_HEADERS_IFD SPIRV-Headers::SPIRV-Headers INTERFACE_INCLUDE_DIRECTORIES)
+    include_directories(${SPIRV_HEADERS_IFD})
+endif()
+
 if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     # Parallel build object files
     add_definitions(/MP)
